faker_edu/__init__.py

```diff
@@ -172,3 +197,26 @@ def faculty_title(self):
         position = self.faculty_position()
         department = self.department_name()
         return position+" of "+department
+
+    def college_name(self):
+        college = self.random_element(['College', 'School'])
+        topic = self.random_element(COLLEGETYPES)
+        name = 'The '
+        if (self.random_element([TRUE, FALSE]) == TRUE):
+            fakeName = faker.providers.person.en_US.provider(Generator())
+            lasts = set(fakeName.last_names.keys())
+            last_name = self.random_element(lasts)
+            """Deciding if the College name will have first and last name, or just last name"""
+            if self.random_element([1, 2]) == 2:
```

Comment:
Please be consistent about how you make a random choice. The pattern from line 205 is preferred.